ycmd/completers/rust/rust_completer.py

```diff
@@ -82,7 +74,7 @@ def _GetResponse( self, handler, request_data = {} ):
     were found.
     """
     self._logger.info('RustCompleter._GetResponse')
-    target = urlparse.urljoin( self._racerd_host, handler )
+    target = 'http://localhost:' + str(self._racerd_port) + str(handler)
```

Comment:
There is a race condition with `utils.GetUnusedLocalhostPort`. Is there a problem with reading the initial output from racerd?